### DISCUSSION TITLE
Question about unit of E0 and continuation of stopped MACE training

### DISCUSSION THREAD

**Discussion by turbosonics** [ACEsuit/mace#413] [Q&A] [answered]
1. May I ask about the unit of E0? 

I performed VASP SCF of a single H atom in 15x15x15 cubic angstrom using the same conditions and pseudopotentials with my other VASP AIMD setting. I get -.11279928E+00 eV = -10.88348 kJ/mol = -2.601213 kcal/mol. 

But when I see the example, I can see:  
`--E0s='{1:-13.663181292231226, 6:-1029.2809654211628, 7:-1484.1187695035828, 8:-2042.0330099956639}'`
I think that means the energy of H atom is -13.6632, right? Is this kJ/mol? 

So... which value should I need to write for my E0 setting? 10.88348? or -0.11279928? 

2. How to continue stopped MACE training job? 
Our local server cluster has a wall time limit of 7 days. Some of my MACE training jobs with big training set is killed without reaching the target epoch.  

In this case, how can I continue the stopped MACE training? Also, is there anything I need to do or be careful when I continue the halted training job? Like, I wonder if CPU training should be continued to CPU and GPU training should be continued from GPU... 

Thanks

**Comment by bernstei** [comment]
E0s are the same units as all the other energies, so eV by default.

Those example E0s are huge for VASP, but plausibly consistent with the way CASTEP does things. VASP gets E0s of single digit eV values.

**Reply by turbosonics** [reply]
Thank you! I will use eV value. Indeed, I was a bit surprised when I check E0 value and compared to the example. 

Also, regarding continuation (or restart), can I just have "--restart_latest" in the input and submit the job in the same folder?

**Comment by gabor1** [comment, marked as answer]
Just to expand on @bernstei 's reply. The units of energy need to be eV (which is coupled to the unit of forces needing to be eV/Å and the units of distance in the structures need to be Å). But the actual values depend on the electronic structure method that you generate the data with. The recommendation is that you design your potential to fit the atomization energy (i.e. the difference between isolated atom energies and the condensed phase energy), so that E0s should be something like the energy of each element as a single atom in vacuum. That's why it depends on what electronic structure method you use, whether you use pseudopotentials or not, etc. Note that you are are not compelled to use the isolated atom energy precisely, e.g. you could decide to keep the isolated atom energy spin-unpolarised, which would be wrong with respect to the reality of an isolated atom, but perhaps easier to fit (as long as you are not too bothered by the error with respect to reality the model will make for the isolated atom). MACE is designed to reproduce the E0 of the isolated atom analytically when you evaluate it on an isolated atom. Hope this helps.

**Reply by turbosonics** [reply]
Thank you so much for detailed explanation.
